Models/Movement.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace BD_oneLove.Models
+{
+    internal class Movement
+    {
+
+        #region Props
+
+        public string Id { get; set; }
+
+        public string StudentId { get; set; }
+        public string StudentFIO { get; set; }
+
```

Comment:
Maybe it is easier just to save Student s ?

Reply:
Well, I thought about that. And probably it would be more correct. But with out structure this change implies either copy-pasting code from GetStudent method in Data Storage or invoking new connection, when calling this method. So I though it'll be easier for now just to save id and FIO. Perhaps, later I will change it.

Reply:
ok